packages/editor/src/client/blocks/mapLeafletBlock.tsx

```diff
@@ -0,0 +1,169 @@
+import React, {useState} from 'react'
+import {Map, TileLayer, Marker} from 'react-leaflet'
+import 'leaflet/dist/leaflet.css'
+import '../customCSS/leaflet.css'
+
+import {Icon} from 'leaflet'
+
+// workaround to ensure that leaflet's images work after going through webpack
```

Comment:
We should clean this up before the merge

Reply:
Done